app/constants/mods.py

```diff
@@ -80,10 +80,7 @@ def convert_str(cls, mods: str) -> Mods:
     def rankable(self) -> bool:
         """Checks if the mod combo is rank-worthy."""
 
-        if self & Mods.AUTOPLAY:
-            return False
-
-        return True
+        return self not in UNRANKABLE_MODS
```

Comment:
due to modes being bitwise, i’m sure this won’t work

Reply:
Good catch, forgot to consider that mods are bitwise.

Reply:
Corrected